app/services/activity/import.rb

```diff
@@ -219,7 +256,17 @@ def create
         ) { |a|
           a.form_state = "complete"
         }
-        @activity.assign_attributes(@converter.to_h)
+
+        attributes = @converter.to_h
+
+        if @is_oda == false
+          incorrect_oda_type_errors = Activity::Import.incorrect_oda_type_errors(@activity, attributes)
+          @errors.merge!(incorrect_oda_type_errors)
+        end
+
+        return unless @errors.blank?
+
+        @activity.assign_attributes(attributes)
```

Comment:
If comparing this with the changes in `ActivityUpdater` above, you can see this isn't very DRY (except the `Activity::Import.incorrect_oda_type_errors` abstraction), but neither is some of the other code in these methods, so I've added a tech debt card